roles/ansible-test/tasks/init_collection.yaml

```diff
@@ -14,15 +14,26 @@
   shell: "cat galaxy.yml | {{ ansible_test_venv_path }}/bin/yq -y .name | tail -n +1 | head -1"
   register: _collection_name
 
-- name: Setup location of project
+- name: Setup location of project for integration tests
+  vars:
+    _test_type: "integration"
   set_fact:
     _test_location: "{{ ansible_test_collection_dir }}/{{ _collection_namespace.stdout }}/{{ _collection_name.stdout }}"
-    _test_cfg_location: "{{ ansible_test_collection_dir }}/{{ _collection_namespace.stdout }}/{{ _collection_name.stdout }}/tests/integration/{{ ansible_test_test_command }}.cfg"
+    _test_cfg_location: "{{ ansible_test_collection_dir }}/{{ _collection_namespace.stdout }}/{{ _collection_name.stdout }}/tests/{{ _test_type }}/{{ ansible_test_test_command }}.cfg"
+  when: ansible_test_test_command == 'network-integration'
+
+- name: Setup location of project for unit tests
+  vars:
+    _test_type: "unit"
+  set_fact:
+    _test_location: "{{ ansible_test_collection_dir }}/{{ _collection_namespace.stdout }}/{{ _collection_name.stdout }}"
+    _test_cfg_location: "{{ ansible_test_collection_dir }}/{{ _collection_namespace.stdout }}/{{ _collection_name.stdout }}/tests/{{ _test_type }}/{{ ansible_test_test_command }}.cfg"
+  when: ansible_test_test_command == 'network-unit'
 
 - name: Install python requirements
   shell: "{{ ansible_test_venv_path }}/bin/pip install -r {{ _test_location }}/requirements.txt -r {{ _test_location }}/test-requirements.txt"
 
 - name: Copy potential cloud provider configuration for ansible-test in the collection
-  shell: "cp -v {{ ansible_test_ansible_path }}/test/integration/cloud-config-*.ini {{ _test_location }}/tests/integration/"
+  shell: "cp -v {{ ansible_test_ansible_path }}/test/{{ _test_type }}/cloud-config-*.ini {{ _test_location }}/tests/{{ _test_type }}/"
   ignore_errors: true
-  when: ansible_test_test_command == 'integration'
+  when: ansible_test_test_command == 'network-integration'
```

Comment:
This is not correct, we only need to do this for 'integration' network-integration doesn't use this file